ci_tendrl_cluster.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Perform following tasks, when applicable
+# (e.g. there are nodes in gluster or ceph_* group)
+# * Configure firewall
+# * Install Tendrl server.
+# * Install Tendrl storage nodes.
+# * Add Gluster repo.
+# * Add Ceph repos.
+- include: qe_pre_installation_tasks.yml
+
+# configure firewall
+- include: firewall_tendrl.yml
+- include: firewall_tendrl_workaround.yml
+
+- include: qe_reboot.yml
+
+- include: tendrl_server.yml
+
+- include: tendrl_node.yml
+
+# Add Centos Gluster YUM repo
+- hosts: gluster
+  remote_user: root
+  roles:
+    - { role: gluster-centos-repo, when: groups.gluster is defined }
+
+# Install centos-release-ceph package with Ceph Repos
+- hosts: ceph_mon:ceph_osd
+  remote_user: root
+  roles:
+  - { role: ceph-centos-repo, when: groups.ceph_osd is defined and groups.ceph_mon is defined }
+
+# workaround according to https://github.com/Tendrl/api/issues/86#issuecomment-285063914
```

Comment:
I am going to test this issue. After that I will give comment if it can be removed.